src/features/Pokedex/PokedexContainer.js

```diff
@@ -11,7 +11,7 @@ function mapStateToProps(state, ownProps) {
   const selectorProps = { id: ownProps.id };
 
   return {
-    errors: getPokedexErrors(state, selectorProps),
+    errors: getPokedexErrorsById(state, selectorProps),
```

Comment:
Also not related to the PR, but using this selector correctly gets the entity `error`s. Before, it was getting the slice `error`s which should only exist when doing collection operations.